Cargo.toml

```diff
@@ -203,3 +204,6 @@ debug         = false     # Set to `true` for debug information
 lto           = "fat"
 opt-level     = 3
 strip         = "symbols" # Set to `false` for debug information
+
+[patch.crates-io]
+filetime = { git = "https://github.com/oligamiq/filetime.git" }
```

Comment:
We should find a stable tag patch to prevent security issues

Reply:
Do we have to manage this by ourselves? This is someone else's code repository

Reply:
In https://github.com/alexcrichton/filetime/issues/109#issuecomment-2708142283:

> My commits is this, but it's old so I forgot it and no test.

we need to test it and contribute to `filetime` crate, the tests can be done via wasmtime like we've done in oxc: https://github.com/oxc-project/oxc/blob/main/.github/workflows/ci.yml#L99